### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by charlesgobina**
## For this Milestone, 
* I created the add book component where users can add new books
* There is a booklist component that shows the list of all the books added and a remove button for each book
* Add and remove functions were implemented.

### CODE REVIEW

File: src/components/booklist.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const removeFromStore = () => {
+    dispatch(removeBook(data2.id));
+  };
+
+  return (
+    <div className="book-list">
+      <ul className="book-items">
+        <li className="d-flex">
+          <span className="title">{ data2.title }</span>
+          <span className="author">{ data2.author }</span>
```
Comment (dasileker) [inline src/components/booklist.js]: - [ ] Please validate your props.

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-  <div className="cat-list">
-    <p>Under Construction</p>
-  </div>
-);
+function Categories() {
+  return (
+    <div className="cat-list">
+      <p>Under Construction</p>
+    </div>
+  );
```
Comment (dasileker) [inline src/redux/categories/categories.js]: - [ ] please verify all the files because sometimes you use the arrow function and some files you don"t.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @charlesgobina ,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 2 above)
Comment (charlesgobina) [reply src/redux/categories/categories.js]: Hello @dasileker,
Thank you for the feedback. I have an issue when validating props, I keep on getting an error on the console saying the prop is validated with "isRequired" but the value is undefined, and it crashes my application. Do you please know how I could walk around that, or any additional resources that will help me out? 

Thank you

**Review by omar-labana — APPROVED** [review]
Hi @charlesgobina, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**